pkg/monitortests/cli/adm_upgrade/status/outputmodel.go

```diff
@@ -260,12 +260,16 @@ func (p *parser) parseControlPlaneOperators() ([]string, error) {
 	var operators []string
 
 	for {
-		line, done := p.next()
+		line, done := p.next(preserveLeadingWhitespace)
 		if done || line == "" {
 			break
 		}
-
-		operators = append(operators, line)
+		trimLeft := strings.TrimLeft(line, " \t")
+		if len(operators) == 0 || line == trimLeft {
+			operators = append(operators, line)
+		} else {
+			operators = append(operators[:len(operators)-1], fmt.Sprintf("%s\n%s", operators[len(operators)-1], trimLeft))
```

Comment:
the append confuses me, this is essentially a rewrite of the last item:

```suggestion
			operators[len(operators)-1] = fmt.Sprintf("%s\n%s", operators[len(operators)-1], trimLeft))
```

and maybe even this way for readability

```
            last := len(operators)-1
			operators[last] = fmt.Sprintf("%s\n%s", operators[last], trimLeft))
```